cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```

Comment:
Let's lose Cypress's example boilerplate (since their comment at the top already has a link to documentation) and add a comment here about how annoying global commands are in Typescript

```suggestion

// NOTE: Custom global `cy.foo()` commands must be added/declared in Typescript
// (@see https://docs.cypress.io/guides/tooling/typescript-support#Types-for-custom-commands)
// which is relatively annoying, so we generally recommend just using utility functions instead.
```

Reply:
I outright deleted commands.js